.travis.yml

```diff
@@ -15,6 +15,12 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 2.0.0
     env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
+  - env: BEAKER_set="ubuntu-12.04-x86_64-openstack"
```

Comment:
This file is automatically generated with modulesync using the informations in metadata.json, so no need to add this here.